core/server/data/schema/default-settings.json

```
@@ -408,8 +408,11 @@
     },
     "unsplash": {
         "unsplash": {
-            "defaultValue": "{\"isActive\": true}",
-            "type": "object"
+            "defaultValue": "true",
+            "validations": {
+                "isIn": [["true", "false"]]
```

Do we want to include the `isEmpty` validation too? 
```suggestion
                "isEmpty": false,
                "isIn": [["true", "false"]]
```

---

Yeah, I think that wouln't hurt and would be more explicit :+1:  Although I'm a bit torn on what difference that would make in most cases where we use `isIn` validation in combination with `isEmpty`. If `isIn` does not specify empty string or `null` as an acceptable value that should mean anything but specified values should fail.